tests/test_ari_text.py

```diff
@@ -761,10 +761,11 @@ def test_ari_text_decode_lit_typed_float32(self):
             ("ari:/REAL32/0.0", 0.0),
             ("ari:/REAL32/-0.", 0.0),
             ("ari:/REAL32/0.255", 0.255),
-            # FIXME: ("ari:/REAL32/0xF", 15.0),
-            # FIXME: ("ari:/REAL32/0xF.", 15.0),
-            # FIXME: ("ari:/REAL32/0xfF", 255.0),
-            # FIXME: ("ari:/REAL32/0xfF.ff", 255.255),
+            # madeline - fix all FIXMEs below for ACE #14
+            ("ari:/REAL32/0xF", 15.0),
```

Comment:
Value segment without the decimal point or exponent separator "p" are not valid for floating point numbers.